ld/zld.xcodeproj/project.pbxproj

```diff
@@ -614,20 +591,11 @@
 				4CDA2DA823FDD2CB00C6009D /* Tweaks.hpp */,
 				4C47258B23FD9E3C00AA02B2 /* MapDefines.h */,
 				F9AA69BF10583E19003E3539 /* Resolver.cpp */,
-				4C8D9C80240580EB0040CE7C /* libLTO-zld.dylib */,
-				4C8D9C82240580EC0040CE7C /* libswiftDemangle-zld.dylib */,
-				4C8D9C81240580EC0040CE7C /* libtapi-zld.dylib */,
 				F9AA69C010583E19003E3539 /* Resolver.h */,
 				F9AA69B410583C0C003E3539 /* SymbolTable.cpp */,
 				F9AA69B510583C0C003E3539 /* SymbolTable.h */,
 				4C5E360823FB61DC0073E2F5 /* compile_stubs.h */,
-				4C22346F240431C2007B3ADE /* libtbb.dylib */,
-				4C223470240431C2007B3ADE /* libtbbmalloc.dylib */,
 				F9AA687A10572E27003E3539 /* InputFiles.cpp */,
-				4C223473240432CC007B3ADE /* libswiftDemangle.dylib */,
```

Comment:
is it no longer linking to libswiftDemangle?

Reply:
This is incorrect, I will add it back